Source/KerbalCombatSystems/DecouplerDesignator.cs

```diff
@@ -1,18 +1,52 @@
-﻿namespace KerbalCombatSystems
+﻿using UnityEngine;
+
+namespace KerbalCombatSystems
 {
     public class ModuleDecouplerDesignate : PartModule
     {
-        const string DecouplerDesignationGroupName = "Decoupler Designation";
+        //denote seperator type
+        [KSPField(isPersistant = true)]
+            public string seperatorType = "seperatorType";
+
 
+        [KSPField(isPersistant = true)]
+            public bool seperated = false;
+
+        const string DecouplerDesignationGroupName = "Seperator Designation";
         [KSPField(
             isPersistant = true,
             guiActive = true,
             guiActiveEditor = true,
-            guiName = "Decoupler Type",
+            guiName = "Seperator Type",
             groupName = DecouplerDesignationGroupName,
             groupDisplayName = DecouplerDesignationGroupName),
             UI_ChooseOption(controlEnabled = true, affectSymCounterparts = UI_Scene.None,
-            options = new string[] { "Default", "Weapon", "Countermeasure", "Escape Pod", "Warhead" })]
-        public string DecouplerType = "Default";
+            options = new string[] { "Default", "Escape Pod", "Warhead" })] //re-add "Countermeasure" at later date
+            public string decouplerDesignation = "Default";
+
+        [KSPEvent(guiActive = true,
+                      guiActiveEditor = false,
+                      guiName = "test",
+                      groupName = DecouplerDesignationGroupName,
+                      groupDisplayName = DecouplerDesignationGroupName)]
```

Comment:
Is this button visible in the right click menu? It would be a duplication of the existing buttons.

Reply:
Testing button, forgot to remove.